services/centralized-grafana/48.3.1/defaults/cm.yaml

```diff
@@ -72,6 +72,8 @@ data:
           allow_loading_unsigned_plugins: "grafana-piechart-panel"
         dashboards:
           default_home_dashboard_path: "/tmp/dashboards/global-overview.json"
+        analytics:
+          reporting_enabled: false
```

Comment:
> check_for_updates
Set to false, disables checking for new versions of Grafana from Grafana’s GitHub repository. When enabled, the check for a new version runs every 10 minutes. It will notify, via the UI, when a new version is available. The check itself will not prompt any auto-updates of the Grafana software, nor will it send any sensitive information.

https://github.com/grafana/helm-charts/blob/bd618deb347d6e756d7964ec2a838b4f1110fd48/charts/grafana/values.yaml#L723-L730 grafana chart also defaults `check_for_updates: true` so we'll also need to explicitly disable it

Reply:
I was considering doing that in another PR -- your call

Reply:
let's just include it in this one so we don't have to have 2 prs per backport!

Reply:
Now in here as well @gracedo